src/webui/src/static/function.ts

```diff
@@ -182,8 +182,6 @@ function formatTimestamp(timestamp?: number, placeholder?: string): string {
 function parseMetrics(metricData: string): any {
     if (metricData.includes('NaN')) {
         return JSON5.parse(metricData)
-    } else {
-        return JSON.parse(metricData)
```

Comment:
why remove this?

Reply:
I thought UI only use JSON5 from the teams message.
It's misleading, I'll change this back.